plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -453,6 +453,7 @@ public void testAggregationPushdown()
         assertThat(query("SELECT count(nationkey) FROM nation")).isFullyPushedDown();
         assertThat(query("SELECT count(1) FROM nation")).isFullyPushedDown();
         assertThat(query("SELECT count() FROM nation")).isFullyPushedDown();
+        assertThat(query("SELECT regionkey, count(1) FROM nation GROUP BY regionkey")).isFullyPushedDown();
```

Comment:
sqlserver too

Reply:
Added the new and some existing missing cases to SqlServer.

Working separately to extract agg pushdown tests to BaseJdbcConnectorTest to avoid the current discrepancy.